improve the content related to self signed docs (#687)
aerogear/mobile-docs: modules/ROOT/pages/_partials/keycloak/setting-up-the-idm-sdk.adoc

NOTE: Initialization also enables authentication.

> If you want to use self-signed certificates with your Cordova application, please follow the instructions link:showcase-apps.html#certs[here].

Reviewer: ```suggestion
To use self-signed certificates with your Cordova application, complete the following instructions link:showcase-apps.html#certs[here].
```

Reviewer: I am manually making this update as part of a new commit.

authService.Configure(authConfig);
----

> If you want to use self-signed certificates with your Cordova application, please follow the instructions link:showcase-apps.html#certs[here].

Reviewer: ```suggestion
To use self-signed certificates with your Cordova application, complete the following instructions link:showcase-apps.html#certs[here].
```

Reviewer: The change was applied elsewhere.
